reference-architecture/vmware-ansible/ocp-on-vmware.py

```diff
@@ -519,7 +523,7 @@ def launch_refarch_env(console_port=8443,
     rhsm_password=%s \
     rhsm_activation_key=%s \
     rhsm_org_id=%s \
-    rhsm_pool=%s \
+    rhsm_pool="%s" \
```

Comment:
@dav1x added the quotes because sub manager was attaching every pool with the word "Red" because of the space between Red Hat